tvb_data/macaque/README

```diff
@@ -0,0 +1,8 @@
+The Virtual Macaque Brain
+-------------------------- 
+  A macaque connectome for large-scale network simulations in TheVirtualBrain
+   Shen, Kelly; Bezgin, Gleb; Everling, Stefan; McIntosh, Anthony R.
+
+--------------------------
+A whole-cortex macaque structural connectome constructed from a combination of axonal tract-tracing and diffusion-weighted imaging data. 
+Created for modeling brain dynamics using TheVirtualBrain platform. Website: thevirtualbrain.org
```

Comment:
Pls enrich a bit the README file:
- put the link to Zenode, from where we tool the original data
- write a few words on what changes we did on that data (describing prepare.py)
- make a summary of what entities can be imported from this folder and in what order in TVB